app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -200,7 +236,7 @@ public boolean onMenuItemClick(MenuItem item) {
     }
 
     private void setColumnNumber(int columns) {
-        subscriptionGridLayout.setNumColumns(columns);
+//        subscriptionGridLayout.setNumColumns(columns);
```

Comment:
Can you call something on the RecyclerView layout manager here to change the columns?